app.yml

```diff
@@ -35,7 +35,7 @@ default_events:
 # - project_card
 # - project_column
 # - public
-- pull_request
+# - pull_request
```

Comment:
why did you need to remove this ? without this you can't put PR's on the boards.

Reply:
I was getting this when registering the bot:

![image](https://user-images.githubusercontent.com/51681/66746164-02fb7c00-ee79-11e9-82e8-601c4fabb4c1.png)

Agreed, we do need to be notified about PRs. I planned to see what was causing this, but ran out of time on Friday.